docs/content/guides/developer/app-examples/e2e-counter.mdx

```diff
@@ -28,18 +28,47 @@ Before getting started, make sure you have:
 
 - [Installed the latest version of Sui](../getting-started/sui-install.mdx).
 
+## Directory structure
+
+To begin, create a new folder on your system titled `react-e2e-counter` to hold all your project files. You can name this directory differently, but the rest of the guide references this file structure. Inside that folder, create two more folders: `move` and `src`. Inside the `move` folder, create a `counter` directory. Finally, create a `sources` folder inside `counter`. Different projects have their own directory structure, but it's common to split code into functional groups to help with maintenance.
+
+:::checkpoint
+
+- You have the latest version of Sui installed. If you run `sui --version` in your terminal or console, it responds with the currently installed version.
```

Comment:
Is there anyway to dynamically display the latest version or let the user know where they can find it? maybe add a link to our [release page](https://github.com/MystenLabs/sui/releases)? Might be overkill.

Reply:
I created this but it grabs the info at build. That means there would be a stretch of time where it was possibly off. Maybe add a clickable element that gets the info on user click...